crate2nix/src/main.rs

```diff
@@ -71,6 +71,14 @@ pub enum Opt {
             help = "The path to the crate hash cache file. Uses 'crate-hashes.json' in the same directory as Cargo.toml by default."
         )]
         crate_hashes: Option<PathBuf>,
+
+        #[structopt(
+            short = "j",
```

Comment:
This strongly implies something about the build since it is like the make argument. I would like to make clear that this is the number of parallel prefetches.